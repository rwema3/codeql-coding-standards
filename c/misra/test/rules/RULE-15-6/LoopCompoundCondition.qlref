rules/RULE-15-6/LoopCompoundCondition.ql