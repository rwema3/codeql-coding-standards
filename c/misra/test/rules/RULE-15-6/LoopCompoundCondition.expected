| test.c:4:3:5:9 | while (...) ... | Loop body not enclosed within braces. |
| test.c:7:3:8:5 | while (...) ... | Loop body not enclosed within braces. |
| test.c:11:3:12:9 | for(...;...;...) ... | Loop body not enclosed within braces. |
| test.c:14:3:15:5 | while (...) ... | Loop body not enclosed within braces. |
